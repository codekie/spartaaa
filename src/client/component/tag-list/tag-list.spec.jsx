import React from 'react';
import TagList from './tag-list.jsx';
import { shallow } from 'enzyme';

it('renders correctly', () => {
    expect(shallow(<TagList tags={['tüdlidü']} />)).toMatchSnapshot();
});
