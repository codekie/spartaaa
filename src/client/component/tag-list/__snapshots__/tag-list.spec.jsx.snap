// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ForwardRef
  className="cmp-tag-list"
  gapless={false}
  style={Object {}}
>
  <ForwardRef
    className=""
    color={null}
    key="t端dlid端"
    onClick={[Function]}
    remove={false}
    renderAs="span"
    rounded={false}
    size={null}
    style={Object {}}
  >
    t端dlid端
  </ForwardRef>
</ForwardRef>
`;
